ios/AppLovinMAXAdView.m

```diff
@@ -295,7 +295,11 @@ - (void)didMoveToWindow
     [super didMoveToWindow];
     
     // This view is unmounted
+#ifdef RCT_NEW_ARCH_ENABLED
+    if ( !self.window )
+#else
     if ( !self.window && !self.superview )
+#endif
```

Comment:
Have you tested if updating to `if ( !self.window )` for old/new arch would work? Instead of having this fork in logic.

Reply:
I've tested.  `( !self.window )` works for both old and new arch.  I'll update without the preprocessor.

Reply:
Cool, less code is more, thank you.